### DISCUSSION TITLE
First time trying to use Vercel .. account blocked!?

### DISCUSSION THREAD

**Discussion by cummins-orgs** [vercel/community#3783] [Account Issue] [answered]
### Question

Why would my account be blocked before even signing up with the service? 

highest friction UX I've experienced for a while, lol.

**Comment by aronnihc** [comment]
same here, trying out vercel for the first time but then after clicking [Continue with GitHub] it says user account blocked for no reason but still receiving emails from vercel

**Reply by cummins-orgs** [reply]
I just set up on netlify instead, works fine. Watch this space as my app develops! https://bill-the-scheduler.netlify.app/

**Comment by amyegan** [comment, marked as answer]
Sorry you're having ~~login~~ registration issues 🙁 

Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy.

**Comment by cummins-orgs** [comment]
Neither of us had 'login issues'... we never succeeded in registering for first use. Obviously Vercel is within its rights to deny service, but I don't think that's a fair description of the issue.

**Reply by amyegan** [reply]
I know it's frustrating. I really can't ask you about account details here because of safety and security concerns. The marked answer has instructions for what you can do next to solve the problem. I did update the answer for you so it says "registration" instead of "login", though the next steps are the same in either case.
